controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -15,7 +15,7 @@ const InitDiscoveryDomainData = async (req, res) => {
   try {
     let suggestedDomainsQuery = `SELECT 
                 C.domain_name, C.short_description, C.logo, C.credder_score,
-                A.domain_id_followed, 
+                C.domain_page_id AS domain_id_followed, 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The alias for `domain_id_followed` has been changed to `C.domain_page_id`. This change could potentially cause issues if the rest of the code expects the alias to be `domain_id_followed`. Please ensure that all references to this column in the code are updated accordingly. If not, it might lead to undefined variables or properties in your code.
```diff
-                A.domain_id_followed, 
+                C.domain_page_id AS domain_id_followed, 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
It changed as requested by mas Yusuf to standardized the response in FE